spec/myapp_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../myapp'
+require 'rspec'
+require 'rack/test'
+
+describe 'Server Service' do
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application
+  end
+
+  context 'GET to /' do
+    let(:response) { get '/' }
+    it 'returns status 200 OK' do
```

Comment:
Espaço entre as declarações

Reply:
Olá Bruno. Eu não consegui entender onde estão faltando os espaços.

Reply:
@feperessim outro detalhe, o let é utilizado como se fosse uma variável para armazenar dados que serão utilizados no teste, ou mesmo constantes.

https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let

No caso ai, vc confundiu com o `subject` de uma lida nos 3 items aqui:
https://relishapp.com/rspec/rspec-core/docs/subject

subject(:response) { get '/' }